utils/log-target.js

```diff
@@ -27,6 +27,25 @@ const CDN_REGEX = new RegExp(
     '\\.ampproject\\.net/',
   'i'
 );
+const CHANNEL_TYPES = {
```

Comment:
I notice these are mostly the same as https://github.com/ampproject/error-tracker/blob/7f607b3ba9f3cdaa427e2d6ef23ac9e3f08e3051/utils/human-rtv.js#L16-L32, with a few duplicate names and changed names.

Could we just reuse the same mapping? I would actually prefer to not mix Canary and RC, since we test _really_ experimental features in Canary.

Reply:
I imagine your concern is being able to detect errors in Experimental features, and separate them out from the errors in Beta. With this grouping, that won't be a problem. My thinking around this is that the version names will already differentiate between Beta/Experimental. A single channel (ie. Experimental) can also be viewed using the built-in filtering mechanisms.

My thinking around not using the same mapping is twofold
1. It becomes redundant, as each service bucket will contain only the release matching its channel (with the exception of opt-in and percentage, which would be identical); this means we'd still have the giant list of services. Since older RTVs quickly drop off the list, that makes each service bucket contain only one or two versions, and we lose the benefit that service grouping provides.
2. By grouping in this way, each service represents a specific commit in the code. Everything in Production is running from the same point, but with different configs; likewise for Development, Nightly, etc. The version dropdown then provides further refinement. By grouping Stable+Control, or Beta+Experimental, I think we'll get better signal from both, while maintaining the same ability to narrow down the list as desired.

WDYT?

Reply:
> My thinking around this is that the version names will already differentiate between Beta/Experimental. A single channel (ie. Experimental) can also be viewed using the built-in filtering mechanisms.

Excellent, this was my concern.

> It becomes redundant, as each service bucket will contain only the release matching its channel (with the exception of opt-in and percentage, which would be identical); this means we'd still have the giant list of services.

Is this level of detail even needed for the service name? Perhaps we should just group by `CDN` and `Prod`ness, and use the version level filtering?

If we do stick with this list, lets extract it and the one from `human-rtv.js` into a common file.

Reply:
Done, and then collected the various RTV utils into one folder.